tests/src/Unit/OgAccessTest.php

```diff
@@ -64,26 +69,4 @@ public function testUserAccessOwner($operation) {
     $this->assertTrue($user_access->isAllowed());
   }
 
-  /**
-   * @coversDefaultmethod ::userAccess
-   * @dataProvider operationProvider
-   */
-  public function testUserAccessOgUserAccessAlter($operation) {
-    $permissions[OgAccess::ADMINISTER_GROUP_PERMISSION] = TRUE;
-    \Drupal::getContainer()->set('module_handler', new OgAccessTestAlter($permissions));
-    $group_entity = $this->groupEntity();
-    $group_entity->id()->willReturn(mt_rand(5, 10));
-    $user_access = OgAccess::userAccess($group_entity->reveal(), $operation, $this->user->reveal());
-    $this->assertTrue($user_access->isAllowed());
-  }
```

Comment:
it's actually not hook - is about changing a bit the mocking of the internal cache. As I want to get this PR soon, I thought it would make sense to create a follow up